tests/unicon/Makefile

```diff
@@ -0,0 +1,21 @@
+
+include ../Makedefs
+
+TARGETS=$(patsubst %.icn,%,$(wildcard *.icn))
+
+SKIP= tester FxPtTests\
+      audio dbi list_test nlm q rec test testnlm clin_err hello mintest ovld q2 sel to
+
+Test: DoTest
+
+include ../Makefile.test
+
+FxPtTest: FxPtTest.icn tester.u
+
+FxPtTest_OVLD: FxPtTest_OVLD.icn tester.u
+
+tester.u: tester.icn
+	$(UC) -s -u -c $^
```

Comment:
The test is failing because `UC` relative path is wrong from this directory.

Reply:
But since it is also wrong for _every other test_ in the test suite -- I guessed that the problem is in the Debian-Pkg configuration, not this PR.

Reply:
Yes, that is a separate issue, but as you can see, the tests above ignore. And only your PR fails the tests. Only  the new Makefile attempts to compile the test in the subfolder.